source/getting-started/core-concepts.md

```diff
@@ -1,82 +1,59 @@
-change, Ember automatically keeps the URL in the browser's address bar
-up-to-date.
-
-This means that, at any point, users are able to share the URL of your
-app. When someone clicks the link, they reliably see the same content as
-the original user.
-
-#### Components
-
-A **component** is a custom HTML tag whose behavior you implement using
-JavaScript and whose appearance you describe using Handlebars templates.
-They allow you to create reusable controls that can simplify your
-application's templates.
-
-#### Models
-
-A **model** is an object that stores _persistent state_. Templates are
-responsible for displaying the model to the user by turning it into
-HTML. In many applications, models are loaded via an HTTP JSON API,
-although Ember is agnostic to the backend that you choose.
-
-
-#### Routes
-
-A **route** is an object that tells the template which model it should
-display.
-
-These are the core concepts you'll need to understand as you develop
-your Ember.js app. They are designed to scale up in complexity, so that
-adding new functionality doesn't force you to go back and refactor major
-parts of your app.
-
-Now that you understand the roles of these objects, you're equipped to
-dive deep into Ember.js and learn the details of how each of these
-individual pieces work.
+## Templates
+
+Templates, written in the Handlebars language, describe the user interface of
+your application. In addition to plain HTML, templates can contain expressions,
+like `{{title}}` or `{{author}}`, which take information from a component and
+put it into HTML. They can also contain helpers, like `{{if isAdmin}}30 people
+have viewed your blog today.{{/if}}.` Finally, they can contain components, such
+as if a template listing out all the blog posts in an app rendered a component
+for each post.
+
+## Components
+
+Components are the primary way user interfaces are organized in Ember. They
+consist of two parts: a template, and JavaScript that defines the component's
+behavior. For example, a blog application might have a component for displaying
+a list of blog posts called `all-posts`, and another component for displaying an
+individual post called `view-post`. If users can upvote a post, the `view-post`
+component might define a behavior like _when the user clicks the upvote button,
+increase the `vote` property's value by 1`_.
+
+## Controllers
+
+Controllers are very much like components, so much so that in future versions of
+Ember, controllers will be replaced entirely with components. At the moment,
+components cannot be routed to (see below), but when this changes, it will be
+recommended to replace all controllers with components.
+
+## Models
+
+Models stores _persistent state_. For example, a blog application would want to
+save the content of a blog post when a user publishes it, and so the blog post
+would have a model defining it, perhaps called the `Post` model. A model
```

Comment:
I like that wording. I'm not sure about including a link, though, because this is meant to be the 30-second overview. They'll get to the full explanation of all these concepts in the relevant sections.